### ISSUE TITLE
Use NetworkLayout.jl for the algorithms

### ISSUE THREAD

**Issue by abhijithanilkumar** [abhijithanilkumar/NetworkViz.jl#13]
Modifying the package to use NetworkLayout.jl so that redundant copied code from GraphLayout can be eliminated.

**Comment by hpoit** [comment]
Hi Abhijith, awesome work. Will solving this issue also make available layered graph drawing with the Buchheim Tree Drawing of NetworkLayout.jl?

**Comment by hpoit** [comment]
I'm sorry, trees are a special case of DAGs, so no, it is not general enough. I'm thinking of adding layered graph drawing to your package, to be used in BayesNets.jl.

**Comment by abhijithanilkumar** [comment]
Yeah, Buchheim can be made available here, but for that to work, a cycle removal function needs to be added as mentioned [here](https://github.com/JuliaGraphs/NetworkLayout.jl/issues/9). Adding new algorithms here is not a great idea since NetworkLayout was created to separate layouting code from visualizing code.
